crates/tracing/src/lib.rs

```diff
@@ -3,6 +3,10 @@
 #[cfg(feature = "otel")]
 pub mod context;
 
+pub mod metrics;
+
+pub use metrics::configure_metrics;
```

Comment:
Nit: it feels weird to have metrics be a sub-module but re-exported here. Shouldn't tracing and metrics be "peers"?

Reply:
talked offline, opting to leave this for the future PR that reworks these under a more cohesive observability umbrella.